### PR TITLE
Update 0407.lang

### PR DESCRIPTION
**Pull request by speedy-pc**
Translation improved, text in the description formulated in a more understandable manner. 
DE: Übersetzung verbessert, Text in der Beschreibung verständlicher formuliert.

### CODE REVIEW

**Comment by micwoj92** [comment]
@botflakes @JimBim89 @schmelto @htcfreek

File: Langs/0407.lang [hunk 1]
```diff
@@ -27,11 +27,11 @@ RAM Installed=Installierter RAM
 Secure Boot=Secure Boot
 Storage Available=Verfügbarer Speicher
 TPM Version=TPM-Version
-Checking...=überprüfe...
-64 Bit CPU=64-Bit CPU
-64 Bit OS=64-Bit Betriebssystem
-32 Bit CPU=32-Bit CPU
-32 Bit OS=32-Bit Betriebssystem
+Checking...=überprüfe, bitte warten
```
Comment (schmelto) [inline Langs/0407.lang:30]:
```suggestion
Checking...=überprüfe, bitte warten...
```
suggesting adding the full stops

File: Langs/0407.lang [hunk 2]
```diff
@@ -70,13 +70,13 @@ Loading WMIC=Lade WMIC
...
-CPU Name=Die CPU, die in Ihrem Computer installiert ist. Die Windows 11 Kompatibilität kann sich noch ändern. Bei Inkompatibilität, muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
-CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Bei Inkompatibilität, muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
-CPU Speed=Die Geschwindigkeit, mit der Ihre CPU-Aufgaben verarbeitet. Bei Inkompatibilität, muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+Boot=Eine Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. "Legacy" bedeutet, dass sich ihr System im traditionellen Startmodus befindet. Dies kann in den BIOS/UEFI-Einstellungen korrigiert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
+CPU Name=Die Prozessor, der in Ihrem Computer verbaut ist. Die Windows 11 Kompatibilität kann sich noch ändern. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
```
Comment (schmelto) [inline Langs/0407.lang:74]:
```suggestion
CPU Name=Der Prozessor, der in Ihrem Computer verbaut ist. Die Windows 11 Kompatibilität kann sich noch ändern. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
```

**Review by schmelto — COMMENTED** [review]
Those are my sugestions otherwise its looking good for me

File: Langs/0407.lang [hunk 3]
```diff
@@ -70,13 +70,13 @@ Loading WMIC=Lade WMIC
...
+Disk Type=Der Partitionsstil Ihrer Festplatte oder SSD. Windows 11 benötigt eine GPT-Partition. Sollten Sie MBR anstatt GPT nutzen, können Sie dies mit dem MBR2GPT-Tool von Microsoft korrigieren.
+RAM=Die Größe des in Ihrem Computer installierten Arbeitsspeichers. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+Secure Boot=Secure Boot (sicherer Start) ist eine Funktion innerhalb des UEFI-BIOS. Für Windows 11 muss diese Funktion vorhanden sein. Wenn Secure Boot nicht erkannt wird, kann dies bei neueren PCs oder Laptops im UEFI-BIOS aktiviert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
+Storage=Der Speicherplatz Ihrer Festplatte oder SSD. Eine Aufrüstung ist bei Desktop PCs sowie vielen Laptops meistens möglich.
+TPM=Trusted Platform Module (TPM) ist ein Sicherheitsmodul des Computers, dass von Windows 11 benötigt wird. Nicht immer ist ein separates TPM-Modul notwendig. Viele moderne CPUs enthalten diese Funktion. Die Aktivierung von TPM ist oft in den UEFI-BIOS-Einstellungen möglich. Im BIOS nennt sich diese Funktion "fTPM" (AMD) oder "PTT" (Intel). Schlagen Sie im Handbuch Ihres Motherboards nach.
```
Comment (JimBim89) [inline Langs/0407.lang:82]:
```suggestion
TPM=Trusted Platform Module (TPM) ist ein Sicherheitsmodul des Computers, das von Windows 11 benötigt wird. Nicht immer ist ein separates TPM-Modul notwendig. Viele moderne CPUs enthalten diese Funktion. Die Aktivierung von TPM ist oft in den UEFI-BIOS-Einstellungen möglich. Im BIOS nennt sich diese Funktion "fTPM" (AMD) oder "PTT" (Intel). Schlagen Sie im Handbuch Ihres Motherboards nach.
```

File: Langs/0407.lang [hunk 4]
```diff
@@ -70,13 +70,13 @@ Loading WMIC=Lade WMIC
...
-CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Bei Inkompatibilität, muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
-CPU Speed=Die Geschwindigkeit, mit der Ihre CPU-Aufgaben verarbeitet. Bei Inkompatibilität, muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+Boot=Eine Boot-Methode, die Ihr Computer zum Starten verwendet. Windows 11 benötigt den UEFI-Modus. "Legacy" bedeutet, dass sich ihr System im traditionellen Startmodus befindet. Dies kann in den BIOS/UEFI-Einstellungen korrigiert werden. Schlagen Sie im Handbuch Ihres Motherboards nach.
+CPU Name=Die Prozessor, der in Ihrem Computer verbaut ist. Die Windows 11 Kompatibilität kann sich noch ändern. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+CPU Cores=Die Anzahl der Rechenkerne, die Ihrer CPU zur Verfügung stehen. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
+CPU Speed=Die Geschwindigkeit, mit der Ihre CPU-Aufgaben verarbeitet. Bei Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
```
Comment (JimBim89) [inline Langs/0407.lang:76]:
```suggestion
CPU Speed=Die Geschwindigkeit, mit der Ihre CPU-Aufgaben verarbeitet. Bei einer Inkompatibilität muss die CPU ersetzt werden. Dies ist bei vielen Laptops nicht möglich.
```

**Review by JimBim89 — COMMENTED** [review]
Fixed inconsistency and typo.

File: Langs/0407.lang (hunk 3 above)
Comment (schmelto) [reply Langs/0407.lang:82]: I think `dass`  is correct here or not?

File: Langs/0407.lang (hunk 2 above)
Comment (htcfreek) [reply Langs/0407.lang:74]:
```suggestion
CPU Name=Der Prozessor, welcher in Ihrem Computer verbaut ist. Die Liste der zu Windows 11 kompatiblen Prozessoren kann sich noch ändern. Bei einer Inkompatibilität muss der Prozessor ersetzt werden. Achtung: Dies ist bei vielen Laptops nicht möglich.
```

File: Langs/0407.lang (hunk 3 above)
Comment (htcfreek) [reply Langs/0407.lang:82]: If it could be replaced with `wepches`, you need `das`!

File: Langs/0407.lang (hunk 4 above)
Comment (htcfreek) [reply Langs/0407.lang:76]: **Better would be:**

```suggestion
CPU Speed=Die Geschwindigkeit, mit der Ihre CPU die Aufgaben abarbeitet. Bei einer Inkompatibilität muss die CPU ersetzt werden. Achtung: Dies ist bei vielen Laptops nicht möglich.
```

**Review by htcfreek — CHANGES_REQUESTED** [review]
👉 See my comments on the other reviews before.

File: Langs/0407.lang (hunk 3 above)
Comment (schmelto) [reply Langs/0407.lang:82]: > If it could be replaced with `welches`, you need `das`!

😅 Ups

Comment (micwoj92) [reply Langs/0407.lang:82]: Should this change also be commited or not?

**Pull request merged**